cirq/ops/gate_features.py

```diff
@@ -50,7 +50,7 @@ def on_each(self, *targets: Union[raw_types.Qid, Iterable[Any]]
         Raises:
             ValueError if targets are not instances of Qid or List[Qid].
         """
-        operations = []  # type: List[raw_types.Operation]
+        operations: raw_types.Operation = []
```

Comment:
Why change this in this PR?  Also missing List[...].

Reply:
fixed.  I slightly prefer not using the comments